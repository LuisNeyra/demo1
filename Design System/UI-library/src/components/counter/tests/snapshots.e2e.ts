import { snapIt } from '../../../test';

const component = 'counter';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('a default empty counter', `<blaze-counter></blaze-counter>`);
  });
});
