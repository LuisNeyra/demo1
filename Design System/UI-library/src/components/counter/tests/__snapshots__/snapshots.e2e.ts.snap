// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`counter renders a default empty counter 1`] = `"<blaze-counter class=\\"hydrated\\"><!----><div class=\\"c-counter\\"><span class=\\"c-counter__prefix\\"></span><span class=\\"c-counter__amount\\"></span><span class=\\"c-counter__suffix\\"></span></div></blaze-counter>"`;
